src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/lib/get_indices.test.ts

```diff
@@ -19,7 +19,8 @@
 
 import { getIndices } from './get_indices';
 import { IndexPatternCreationConfig } from './../../../../../../../management/public';
-import { LegacyApiCaller } from '../../../../../../../../../plugins/data/public';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { LegacyApiCaller } from '../../../../../../../../../plugins/data/public/search';
```

Comment:
I changed the import to an internal one here, as this should go away anyway.
I will do a cleanup of other unnecessary imports in follow up PRs as we move the rest of the search \ agg config code into the plugin.